modules/asset_import/0002-alembic-1.7.9-remove-msc_ver-and-fix-builtins-and-includes.patch

```diff
@@ -0,0 +1,218 @@
+diff -Naur alembic-1.7.9.orig/bin/AbcLs/AbcLs.cpp alembic-1.7.9/bin/AbcLs/AbcLs.cpp
+--- alembic-1.7.9.orig/bin/AbcLs/AbcLs.cpp	2018-10-12 17:05:36.907226500 -0400
++++ alembic-1.7.9/bin/AbcLs/AbcLs.cpp	2018-10-12 17:07:03.914695000 -0400
+@@ -52,11 +52,11 @@
```

Comment:
Are you aware that you're submitting a patch within a patch?

Reply:
These are the changes to upstream alembic source. If I were to update alembic, it makes updating easier because I have a copy of the exact changes outside of git.